recaptcha_enterprise/snippets/src/test/java/app/SnippetsIT.java

```diff
@@ -193,7 +193,6 @@ public void testCreateAnnotateAccountDefender()
         "HmacSHA256");
     mac.init(secretKeySpec);
     byte[] hashBytes = mac.doFinal(accountId.getBytes(StandardCharsets.UTF_8));
```

Comment:
This code seems to be unused (line 188 to 195). Can that be removed as well?

Reply:
Oh yeah. Sorry, missed that. Done.